### PR TITLE
deps: update sipi to 3.6.3

### PR DESCRIPTION
**Pull request by subotic**
### Task Description/Number

Issue Number: DEV-1632

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] Bugfix: represents bug fixes
- [ ] Refactor: represents production code refactoring
- [ ] Feature: represents a new feature
- [ ] Documentation: documentation changes (no production code change)
- [ ] Chore: maintenance tasks (no production code change)
- [ ] Style: styles updates (no production code change)
- [ ] Test: all about tests: adding, refactoring tests (no production code change)
- [x] Other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: webapi/src/it/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala [hunk 1]
```diff
@@ -847,12 +839,7 @@ class KnoraSipiIntegrationV1ITSpec
         baseApiUrl + "/v1/resources/" + URLEncoder.encode(pdfResourceIri.get, "UTF-8")
       ) ~> addCredentials(BasicHttpCredentials(userEmail, password))
 
-      val documentResourceResponse: JsObject = getResponseJson(documentResourceRequest)
-      val locdata                            = documentResourceResponse.fields("resinfo").asJsObject.fields("locdata").asJsObject
-      val nx                                 = locdata.fields("nx").asInstanceOf[JsNumber].value.toInt
-      val ny                                 = locdata.fields("ny").asInstanceOf[JsNumber].value.toInt
-      assert(nx == testPdfWidth)
-      assert(ny == testPdfHeight)
+      getResponseJson(documentResourceRequest)
```
Comment (irinaschubert) [inline webapi/src/it/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala:842]: I don't understand what this test is actually testing. Shouldn't there at least be something to assert, e.g. the new filename?

File: webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala [hunk 2]
```diff
@@ -3259,22 +3259,10 @@ case class DocumentFileValueContentV2(
...
-        val maybePageCountStatement: Option[(IRI, JsonLDInt)] = pageCount.map { definedPageCount =>
-          OntologyConstants.KnoraApiV2Complex.DocumentFileValueHasPageCount -> JsonLDInt(definedPageCount)
-        }
-
         JsonLDObject(
           toJsonLDObjectMapInComplexSchema(
             fileUrl
-          ) ++ maybeDimXStatement ++ maybeDimYStatement ++ maybePageCountStatement
+          )
         )
```
Comment (irinaschubert) [inline webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala:3266]: Could this be put into one line?

**Review by irinaschubert — APPROVED** [review]
LGTM, would it be possible to add the change log to the new Sipi release here https://github.com/dasch-swiss/sipi/releases/tag/3.6.3 (like in the other Sipi releases)? Otherwise it's hard to follow what has changed.

**Review by seakayone — APPROVED** [review]

File: webapi/src/it/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala (hunk 1 above)
Comment (subotic) [reply webapi/src/it/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala:842]: Good point, I'm not sure myself.

File: webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala (hunk 2 above)
Comment (subotic) [reply webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala:3266]: Yes, It could. But I'm not sure if it will be more readable when everything is put on one line.
